openedx/core/djangoapps/user_api/management/commands/email_opt_in_list.py

```diff
@@ -255,6 +256,15 @@ def _write_email_opt_in_prefs(self, file_handle, org_aliases, courses):
         row_count = 0
         for row in self._iterate_results(cursor):
             user_id, username, email, full_name, course_id, is_opted_in, pref_set_datetime = row
+
+            if pref_set_datetime:
```

Comment:
@bmedx @doctoryes Lets pick a Todo tag or a standard comment to mark these shims with so they will be easy to find for removal after the 1.11 upgrade is complete.

Reply:
cc @pwnage101

Reply:
Agreed - my only request is that it's unique and easily searchable.